middlewares/no_auth_required.js

```diff
@@ -1,3 +1,6 @@
-/***
- * @todo Redirect the user to main page if token is present.
- */
\ No newline at end of file
+
+export function Authorized(token){
+ if (token) {
+    console.log("Welcome Back!");
```

Comment:
You should have redirected the already logged In user to the index page

Reply:
I redirected user to homepage now by changing auth_required.js and no_auth_required.js. Please, review now!

Reply:
Sorry for the delay in making changes due to endsems.